pkgs/development/python-modules/django-organizations/default.nix

```diff
@@ -0,0 +1,38 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  hatchling,
+  django,
+  django-extensions,
+}:
+
+buildPythonPackage rec {
+  pname = "django-organizations";
+  version = "2.5.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "bennylope";
+    repo = "django-organizations";
+    tag = version;
+    hash = "sha256-lgri6CCITp1oYCwpH8vrUglphXOmwZ3KX5G/L29akrs=";
+  };
+
+  build-system = [ hatchling ];
+
+  dependencies = [
+    django
+    django-extensions
+  ];
+
+  pythonImportsCheck = [ "organizations" ];
+
```

Comment:
Same here

Reply:
Testing this would require setting up a full django environment